tests/acceptance/16_cf-serverd/add_access_constraints_fail.cf

```diff
@@ -0,0 +1,51 @@
+body common control
+{
+  inputs => { "../default.cf.sub" };
+  bundlesequence => { default("$(this.promise_filename)") };
+}
+
+bundle agent init
+{
+  # gdb version must be >= 7 for the gdb scripts to work
+  meta:
+    "test_skip_needs_work" string => "redhat_4|debian_4|aix|debian_6|hpux";
```

Comment:
This needs a meta attribute attached to it referencing the jira issue.
I think it may not skip properly without the issue reference.

     "test_skip_needs_work" string => "redhat_4|debian_4|aix|debian_6|hpux", meta => { "ENT-3765" };

Reply:
Also if you add a "description" string then the tests print that out when running.

You can see an example of failing output from tests that contain description and soft_fail here: https://tracker.mender.io/browse/CFE-2910

Reply:
Thanks Nick. I'll take a look tomorrow. :)